@ndlib/gatsby-theme-marble/src/components/Shared/ActionButtonGroup/BookmarkGroup/BookmarkLabel/test.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+import { mount } from 'enzyme'
+import BookmarkLabel from './'
+
+test('BookmarkLabel', () => {
+  const wrapper = mount(<BookmarkLabel />)
+  expect(wrapper.find('img').exists()).toBeTruthy()
+  expect(wrapper.find('span').text()).toEqual('Save to a portfolio')
```

Comment:
Opinion: I don't like testing verbiage in unit tests unless it was passed in as a parameter, because it often changes and has no bearing on functionality. This makes your test break even though the component works fine, and the change in verbiage was very likely intentional.